Packages/vcs/Lib/VTKAnimate.py

```diff
@@ -122,8 +122,6 @@ def __init__(self,vcs_self):
         self.renderers = []
         self.last_size = None
         self.modified_listener = None
-        import atexit
-        atexit.register(self.close)
```

Comment:
maybe we can unregister it in the close function? Anyway I will double check the pngs are removed properly. Maybe I will simply use tempfile module instead in order to make sure the pngs are removed.